lib/hawkular/inventory/inventory_api.rb

```diff
@@ -412,9 +412,12 @@ def list_operation_definitions(resource_type_path)
       parsed_path = CanonicalPath.parse(resource_type_path.to_s)
       feed_id = parsed_path.feed_id
       resource_type_id = parsed_path.resource_type_id
-      ret = http_get("/traversal/f;#{feed_id}/rt;#{resource_type_id}/type=ot")
+      ots = http_get("/traversal/f;#{feed_id}/rt;#{resource_type_id}/type=ot")
       res = {}
-      ret.each do |ot|
+      ots.each do |ot|
+        ot_name = ERB::Util.url_encode ot['name']
```

Comment:
In this usage it's basically there to handle spaces in an operation name, so I think it should be fine.